src/devices/xyzroe.ts

```diff
@@ -463,6 +483,117 @@ const definitions: DefinitionWithExtend[] = [
             ]);
         },
     },
+    {
+        zigbeeModel: ['ZigUSB_C6'],
+        model: 'ZigUSB_C6',
+        vendor: 'xyzroe',
+        description: 'Zigbee USB switch with monitoring',
+        fromZigbee: [fz.power_on_behavior, fz.on_off, fzLocal.iasZoneAlarm],
```

Comment:
Both can be replaced with a modernExtend: `onOff` and `iasZoneAlarm`

Reply:
I started with this. But `onOff` doesn't work properly with `power_on_behavior` when there is more than one endpoint. It makes one expose `power_on_behavior` but without specifying endpoint, while `fz` and `tz` uses _N suffix all the time. So `power_on_behavior` option don't work. 

With `iasZoneAlarm` I will try again, OK. But it seems that I also had some unresolvable problem.

Reply:
When I use 
```
    iasZoneAlarm({
      endpointNames: ["1"],
      zoneType: "generic",
      zoneAttributes: ["alarm_1"],
      description: "🔌 Indicates if overcurrent protection is triggered",
    }),
``` 
it doesn't change description. Should I also rework `function iasZoneAlarm(args: IasArgs): ModernExtend {` or I can leave as is?

Reply:
- Does this device support a power on behaviour per endpoint? If yes, we can add support for this in the modernExtend expose
- For iAsZoneAlarm, a new entry should be added to https://github.com/Koenkk/zigbee-herdsman-converters/blob/b53aef5c1d93f33f68835ea96d2902c5a93e8979/src/lib/modernExtend.ts#L1400

Reply:
1. Yes, only the first endpoint. I made a small fix that makes power_on_behaivor work, while multiendpoint (Added suffix, corresponding to the endpoint)

2. Ok, ok i can do it like this. But in my implementation we can pass a unique description. Is it not acceptable?